docs/deploy_streamlit_app.md

```diff
@@ -1,3 +1,156 @@
+
+### Update your app
+
+Your GitHub repository is the source for the app, so that means that any time you push an update to your repo you'll see it reflected in the app in almost real time. Try it out!
+
+Streamlit also smartly detects whether you touched your dependencies, in which case it will automatically do a full redeploy for you - which will take a little more time. But since most updates don't involve dependency changes, you should usually see your app update in real time.
+
+### Collaborate on your app
+
+You can collaborate with others by simply having multiple contributors pushing to the same GitHub repo. If you want to try out something new while still keeping your original app running, just create a new branch, make some changes, and deploy a new version of the Streamlit app.
+
+Every deployed app has its Github source code linked in the "☰" menu on the top right. So if you are looking to understand the code of another Streamlit app, you can simply navigate to the GitHub page from there and read or fork the app.
+
+## App access and usage
+
+Streamlit sharing is completely free and is meant to get you started with sharing your Streamlit apps. If you need a solution with with access controls, ability to deploy from private repos, ability to customize resources, and much more, please check out [Streamlit for Teams](http://streamlit.io/for-teams).
+
+### Access
+
+- Apps are always visible to the entire world.
+- You can only deploy apps that are in a public GitHub repo.
+- Your source code must live in Github. We're looking to expand to other Git hosts soon.
+- Everyone with push access to your repo is automatically a maintainer of the app.
+
+### Resource limits
+
+- You can deploy up to 3 apps per account.
+- Apps get up to 1 CPU, 800 MB of RAM, and 10 GB of dedicated storage in a shared execution environment.
+- Apps do not have access to a GPU.
+- If you have a special good-for-the-world case that needs more resources, [send us an email](mailto:product@streamlit.io) and we'll see about making an exception!
+
+## Managing apps
+
+To view or change your deployed Streamlit apps, use your app dashboard at [share.streamlit.io](https://share.streamlit.io) to view your apps, deploy a new app, delete an app, or reboot an app.
+
+### App dashboard
+
+When you first log into [share.streamlit.io](http://s4a.streamlit.io) you will land your app dashboard, which gives you a list of all your deployed apps. This list does includes apps deployed by other people who have push access to the same repos as you, since you're all managers of those apps. Such apps are indicated with an icon like this one:
```

Comment:
```suggestion
When you first log into [share.streamlit.io](http://s4a.streamlit.io) you will land on your app dashboard, which gives you a list of all your deployed apps. This list does include apps deployed by other people who have push access to the same repos as you, since you're all managers of those apps. Such apps are indicated with an icon like this one:
```